Makefile

```diff
@@ -71,5 +71,8 @@ rhel-release-84: rhel-test-84 release/d2iq-base-RHEL-84$(NAME_POSTFIX)
 rhel-release-86: rhel-test-86 release/d2iq-base-RHEL-86$(NAME_POSTFIX)
 rhel-release: rhel-release-79 rhel-release-84 rhel-release-86
 
+flatcar: $(GOVC)
```

Comment:
testing is also missing here. Which means changes will not verify that flatcar build work like we expect them. We should keep the same quality level for flatcar as for every other distro.